src/main/java/com/neuvector/Scanner.java

```diff
@@ -322,6 +324,15 @@ private static String getMountPath(NVScanner nvScanner){
         return mountPath;
     }
 
+    private static String appendBindMountSharedSuffixIfRequired(final Boolean isBindMountShared, final String mountPath, final Logger log) {
+        String updatedMountPath = mountPath;
+        if(isBindMountShared != null && isBindMountShared){
+            updatedMountPath = new StringBuilder(mountPath).append(":z").toString();
+        }
+        log.debug(updatedMountPath);
```

Comment:
Please add a meaningful message here. Something similar to:

```
log.debug("Using volume binding: {}", updatedMountPath);
```

Reply:
Also, please make it level `info`..

Reply:
Commit: [1c36d0d](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/1c36d0d5f6a06157cea7f2ab47499af639640ef5)